app/views/carriage_types/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= link_to 'Edit', edit_carriage_type_path(@carriage_type) %> |
```

Comment:
Why do you need this symbol at the end of the line ?

Reply:
UX